apps/services/user-notification/src/app/modules/notifications/types.ts

```diff
@@ -1,11 +1,12 @@
 export enum MessageTypes {
   NewDocumentMessage = 'newDocumentMessage',
+  TestMessage = 'testMessage',
 }
 
 export interface Notification {
   messageType: MessageTypes
   title: string
   body: string
-  category: 'NEW_DOCUMENT'
+  category?: string //'NEW_DOCUMENT'
```

Comment:
```suggestion
  category: 'NEW_DOCUMENT' | 'TEST_MESSAGE'
```
Is it vital that `NEW_DOCUMENT` isn't empty in the case of a `newDocumentMessage`?
Wouldn't then the typing be more robust if we allowed another constant, such as `TEST_MESSAGE`, instead of risking category being empty?

Reply:
this is used by the mobile apps - so if this has no value the app opens on the home screen when a user taps on a notification - if the value is something the app supports then it opens that particular screen ... this value is optional and open for further development simply as an optional string ... at least for now